iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/IidmXmlConstants.java

```diff
@@ -15,15 +15,19 @@
 
     public static final String INDENT = "    ";
 
-    private static final List<Integer> VERSION_ARRAY = ImmutableList.of(1, 0);
+    private static final List<Integer> VERSION_ARRAY = ImmutableList.of(1, 1);
 
     public static final String VERSION = VERSION_ARRAY.stream().map(Object::toString).collect(Collectors.joining("."));
 
-    public static final String IIDM_BASE_URI = "http://www.itesla_project.eu/schema/iidm/";
+    public static final String IIDM_BASE_URI = "http://www.powsybl.org/schema/iidm/";
 
     public static final String IIDM_PREFIX = "iidm";
 
-    public static final String IIDM_URI = IIDM_BASE_URI + VERSION_ARRAY.stream().map(Object::toString).collect(Collectors.joining("_"));
+    public static final String IIDM_VERSION_UNDERSCORE = VERSION_ARRAY.stream().map(Object::toString).collect(Collectors.joining("_"));
```

Comment:
To be discussed: is is simpler to define `IIDM_VERSION_UNDERSCORE` to "1_1" and `IIDM_VERSION_POINT` to "1.1"

Reply:
It will mean that we will have to change all of them when releasing a new IIDM XML version. But since they are all in the same file it may not be that